hedera-node/hedera-app/src/main/java/module-info.java

```diff
@@ -48,10 +48,10 @@
     requires io.netty.transport;
     requires org.apache.commons.lang3;
     requires org.apache.logging.log4j;
-    requires org.jetbrains.annotations;
     requires static com.github.spotbugs.annotations;
     requires static com.google.auto.service;
     requires static java.compiler;
+    requires static org.jetbrains.annotations;
```

Comment:
Why are we using `org.jetbrains.annotations` here instead of `edu.umd.cs.findbugs.annotations` that we use everywhere else in services?

The only three files that use it are `RoundStateChangeListener`, `ConcurrentBlockItemWriter`, and `GrpcBlockItemWriter` - all in the package `com.hedera.node.blocks.impl`, and the annotation that's used is `@NotNull` which exists in the usual `edu.umd.cs.findbugs.annotations` module (though it is called `@NonNull` there of course).

(Strangely I don't see `edu.umd.cs.findbugs.annotations` in any `module-info.java` in the repo!  Is it somehow part of the build system?)

Reply:
I was surprised to see this in here too. I only added the `static` part because gradle mandated it

Reply:
I think it was just a mistake in the feature branch - that could be _fixed_ (hint hint)

Reply:
Looks like there are other instances of Jetbrains `@NotNull` in `develop`. I'd prefer to fix this there in a separate PR

Reply:
ok! but before `stream-manager-spike` gets merged to `develop` please!

Reply:
https://github.com/hashgraph/hedera-services/pull/14548